docs/UserGuide.md

```diff
@@ -196,17 +196,35 @@ Format: `edit NAME [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [s/SALARY] [i/INFO]
 
 * Edits the client named `NAME`.
   * `Name` is case-insensitive. E.g. `John` will match `john`.
+  * Spaces in `NAME` will be counted as separate names, unless `NAME` fully matches client's name
+    * Example: `edit John Doe p\88888888` will find clients with names containing `John` and `Doe`,
+                there exists a client with the name `John Doe`
   * The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`.
-  * Only full words will be matched e.g. `Han` will not match `Hans`.
```

Comment:
I think this line is still a valid point to be in the UG

Reply:
Ok added it back in